tests/test_C_integration.py

```diff
@@ -10,7 +10,7 @@ def test_basic_integration() -> None:
 
 
 def test_unique_terms() -> None:
-    terms = unique_terms("sex", "GDC").run(host=host, show_term_count=True)
+    terms = unique_terms(col_name="sex", system="GDC", host=host, show_term_count=True)
```

Comment:
does this require a change to the documentation? If so, I think we should add a ticket for Amanda so she knows what to update in the documentation.

Reply:
This brings the functionality back in line with the documentation.